packages/core/src/app/address/AddressSelectButton.tsx

```diff
@@ -36,8 +36,8 @@ const AddressSelectButton: FunctionComponent<AddressSelectButtonProps & WithLang
     return (
         <a
             aria-controls="addressDropdown"
-            aria-describedby={language.translate('address.enter_or_select_address_action')}
```

Comment:
> The aria-describedby attribute lists the ids of the elements that describe the object. It is used to establish a relationship between widgets or groups and the text that describes them. [[ref](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Reference/Attributes/aria-describedby)]


The `aria-describedby` should reference an `id` rather than providing text content directly.